releasenotes/notes/refactor-aux-operators-79d790f8a693a7c0.yaml

```diff
@@ -0,0 +1,7 @@
+---
+other:
+  - |
+    :func:`qiskit.algorithms.eval_observables` is added that originates from a method previously
```

Comment:
Could you make this a bit more general and say that this is a new convenience method to evaluate observables given a bound circuit? This will end up in the releasenotes and generally visible, so it would be good to formulate it for the broader user audience 🙂